.github/workflows/e2e.yml

```diff
@@ -60,4 +62,4 @@ jobs:
         env:
           E2E_CI_MODE: 'true'
           E2E_CI_TIMEOUT_SEC: '600'
-          E2E_KATHY_ROUNDS: '3'
```

Comment:
I don't think we should need this change with the change to frequency in failures.

Reply:
We don't *need* it, but it's still better IMO. I'm not sure what 3 rounds would catch that 2 won't, and 2 is faster than 3 :)